docs/guides/install-python.md

```diff
@@ -24,17 +24,24 @@ $ uv python install
 
     Python does not publish official distributable binaries. As such, uv uses distributions from the Astral [`python-build-standalone`](https://github.com/astral-sh/python-build-standalone) project. See the [Python distributions](../concepts/python-versions.md#managed-python-distributions) documentation for more details.
 
-Once Python is installed, it will be used by `uv` commands automatically.
+Once Python is installed, it will be used by `uv` commands automatically. uv also adds the installed
+version to your `PATH`:
 
-!!! important
+```console
+$ python3.13
+```
+
+uv only installs a _versioned_ executable by default. To install `python` and `python3` executables,
+include the `--default` option:
```

Comment:
It might be worth briefly saying something about why this is not the default, e.g., your OS may run `python3` and expect it to be the version and environment provided by your OS, or you might be using uv alongside other Python providers, and we don't want to conflict, and whether this is a problem depends a lot on your setup.

We might also want to document how to fix this situation if you're in it - is there something other than `rm`?

Reply:
Hm, I'm not sure if that belongs in the guide, but maybe it'd be worth explaining in the concept page?

> We might also want to document how to fix this situation if you're in it - is there something other than rm?

You can rm it or uninstall and reinstall without the default flag.